examples/gpt2_e2e.py

```diff
@@ -0,0 +1,120 @@
+import os
```

Comment:
We should add some comments to help users understand what the example illustrates.

Reply:
Yes, there will be some document talking about it.